src/logic/map_objects/immovable_program.cc

```diff
@@ -46,8 +46,8 @@ allow workers to interact with an immovable (e.g. a tree will need a "fall" prog
 woodcutters to remove the tree).
 
 It is not mandatory for immovables to define programs. If the immovable defines a program named
-``program``, this program will be started as the main program on creation. Immovables without such a
-program will simply display their main animation indefinitely.
+``main``, this program will be started as the main program on creation. Immovables without such a
+program will simply display their 'idle' animation indefinitely.
```

Comment:
add a note that the name "program" is reserved and deprecated

Reply:
There is a warning message already - should that not be enough?

Reply:
If the documentation says nothing but you get an unexpected warning, people will say that the doc is missing something

Reply:
I have added a note.